benchmark/cluster_containers_setup.py

```diff
@@ -277,87 +366,63 @@ def deploy_and_wait_for_creation(self):
         yield self.deploy()
         yield loop_until(self.reactor,
                          self.is_datasets_deployment_complete,
-                         repeat(1, self.timeout))
+                         repeat(self.wait_interval, self._num_loops))
         yield loop_until(self.reactor,
                          self.is_container_deployment_complete,
-                         repeat(1, self.timeout))
+                         repeat(self.wait_interval, self._num_loops))
 
-    def _build_config(self, ignored):
```

Comment:
Really ugly diff. The build_config function has been deleted.
The create_datasets_and_containers function is a new one. They don't really have code in common.